CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.0.0+2
+
+* Fix computation of authority. This should fix authorization.
```

Comment:
Please bump the patch version... Build metadata is usually only for repacking or typos in comments, readme, pubspec...

https://semver.org/#spec-item-10

Reply:
This actually fixes something :)

Reply:
You are right! Done!